src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Sections/Shipments/Header/_title.html.twig

```diff
@@ -0,0 +1,8 @@
+{% set order = hookable_data.resource %}
+
+<div class="card-title">
+    {{ 'sylius.ui.shipments'|trans }}
+    <span class="ms-2">
+        {{ include('@SyliusAdmin/Order/_Shared/Label/ShippingState/' ~ order.shippingState ~ '.html.twig') }}
+    </span>
+</div>
```

Comment:
I have some considerations here.

1. What is a benefit of keeping it as a separate template. Moving this template in X or Y axis doesn't seem to make sense.
2. If we want to keep it, we might consider moving `.card-title` element to the `_header` template, and split the card title (text) and the state into two separate templates, as moving them in X axis seems reasonable (you might want to add new information or remove existing one).